src/Cynthia.Card.Unity/src/Cynthia.Unity.Card/Assets/Script/LoginScript/Bootstrapper.cs

```diff
@@ -18,7 +18,7 @@ public void Awake()
         var builder = new ContainerBuilder();
         builder.Register(x => DependencyResolver.Container).SingleInstance();
         builder.Register(
-            x => new HubConnectionBuilder().WithUrl($"http://{IP}:5005/hub/gwent", HttpTransportType.WebSockets, options => { options.SkipNegotiation = true; })
+            x => new HubConnectionBuilder().WithUrl($"http://localhost:5005/hub/gwent", HttpTransportType.WebSockets, options => { options.SkipNegotiation = true; })
```

Comment:
Could you please avoid changing this? It is only intended for local testing

Reply:
yes sorry i was sure i will be ignored automatically.  will fix that